src/cli.rs

```diff
@@ -154,79 +154,220 @@ pub struct Cli {
-    /// Fetch the source of `$package $version`
+    // Fetch Commands
+    /// Fetch the source of a package
+    ///
+    /// We will attempt to guess what criteria you want to audit the package for
+    /// based on the current check/suggest status, and show you the meaning of
+    /// those criteria ahead of time.
     #[clap(disable_version_flag = true)]
     Inspect(InspectArgs),
 
     /// Yield a diff against the last reviewed version
+    ///
+    /// We will attempt to guess what criteria you want to audit the package for
+    /// based on the current check/suggest status, and show you the meaning of
+    /// those criteria ahead of time.
     #[clap(disable_version_flag = true)]
     Diff(DiffArgs),
 
-    /// Mark `$package $version` as reviewed
+    // Update State Commands
+    /// Mark a package as audited
+    ///
+    /// This command will do its best to guess what you want to be certifying.
+    ///
+    /// If invoked with no args, it will try to certify the last thing you looked at
+    /// with `inspect` or `diff`. Otherwise you must either supply the package name
+    /// and one version (for a full audit) or two versions (for a delta audit).
+    ///
+    /// Once the package+version(s) have been selected, we will try to guess what
+    /// criteria to certify it for. First we will `check`, and if the check fails
+    /// and your audit would seemingly fix this package, we will use the criteria
+    /// recommended for that fix. If `check` passes, we will assume you are working
+    /// on your backlog and instead use the recommendations of `suggest`.
+    ///
+    /// If this removes the need for an `exemption` will we automatically remove it.
     #[clap(disable_version_flag = true)]
     Certify(CertifyArgs),
 
-    /// Mark `$package $version` as unaudited
+    /// Explicitly regenerate various pieces of information
+    ///
+    /// There are several things that `cargo vet` *can* do for you automatically
+    /// but we choose to make manual just to keep a human in the loop of those
+    /// decisions. Some of these might one day become automatic if we agree they're
+    /// boring/reliable enough.
+    ///
+    /// See the subcommands for specifics.
     #[clap(disable_version_flag = true)]
-    AddUnaudited(AddUnauditedArgs),
+    #[clap(subcommand)]
+    Regenerate(RegenerateSubcommands),
 
-    /// Mark `$package $version` as a violation of policy
+    /// Mark a package as exempted from review
+    ///
+    /// Exemptions are *usually* just "backlog" and the expectation is that you will review
+    /// them "eventually". You should usually only be trying to remove them, but sometimes
+    /// additions are necessary to make progress.
+    ///
+    /// `regenerate exemptions` will do this for your automatically to make `check` pass
+    /// (and remove any unnecessary ones), so we recommend using that over `add-exemption`.
+    /// This command mostly exists as "plumbing" for building tools on top of `cargo vet`.
```

Comment:
If this is a plumbing command, perhaps it should be in the plumbing commands section below?

Reply:
It exists in a twilight between plumbing and procelain, so it sits on the boundary between the two (which otherwise have no user-facing labels, only ordering-by-convention)